File: 7KYU/Odd or Even?/solution.java
public class Codewars {
    public static String oddOrEven (int[] array) {
       var total = 0;
      
       for(int i = 0; i < array.length; i++) {
         total += array[i];
       }
      
       if(total % 2 == 0)  {
         return "even";
       } else {
         return "odd";
       }
         
    }
  }

File: 7KYU/Odd or Even?/solution.c
#include <stddef.h>

const char *odd_or_even(const int *v, size_t sz) {
  int sum = 0;
  
  for(int i = 0; i < sz; i++) {
    sum = sum + v[i];
  }
  
  if(sum % 2 == 0) {
    return "even";
  }
  return "odd";
}